src/content-cli-push.ts

```diff
@@ -32,6 +32,16 @@ class Push {
             .option("-p, --profile <profile>", "Profile which you want to use to push the analysis")
             .option("-a, --pushAnalysis", "Specify this option if you want to push analysis from the CTP file")
             .option("-d, --pushDataModels", "Specify this option if you want to push data models from the CTP file")
+            .option(
+                "--globalPoolName <globalPoolName>",
+                "Specify this option if you want to push all Data models into one newly created pool along with value to set the name of the pool to be created",
+                null
```

Comment:
Why do we need `null` in these two options?

Reply:
@GencBlakqoriP9  It is absolutely valid case if user doesn't specify both of them. And if we don't set the default `null` values - then it will be treated and sent to the backend as `undefined`